src/main/resources/initCitus.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+if [[ $1 == *help || $1 == -h ]]
+then
+  echo "usage: initCitus"
+  echo "\n It is expected that this script is run from conductor-client/src/main/resources"
+  exit 1
+fi
+
+function runPsqlOnNode {
+  sudo docker exec $1 psql -U postgres -c "$2"
+}
+
+function runPsqlFileOnNodeWithOLDB {
+  sudo docker exec $1 psql -U postgres -d openlattice -f $2
+}
+
+function runPsqlFileOnNode {
+  sudo docker exec $1 psql -U postgres -f $2
+}
+
+MASTER_EXTERNAL_PORT=5433 COMPOSE_PROJECT_NAME=citus docker-compose up --scale worker=2 -d
+
+sleep 5
+
+# sudo docker exec citus_master psql -U postgres -c "SELECT * FROM master_get_active_worker_nodes();"
+NODES=`runPsqlOnNode citus_master "SELECT * FROM master_get_active_worker_nodes();"  | grep worker | cut -d ' ' -f 2`
```

Comment:
I pretty much always prefer $() to `` due to nesting issues.

Reply:
This is great. I've never ever gotten feedback on a bash script